### ISSUE TITLE
Needs CI tests for Windows

### ISSUE THREAD

**Issue by TooTallNate** [vercel/fun#7]
Possibly using [CircleCI for Windows beta](https://ideas.circleci.com/ideas/CCI-I-633), or [AppVeyor](https://www.appveyor.com/).

**Comment by MAPESO** [comment]
@TooTallNate Our best option for now is to use `AppVeyor`. Circle CI it still does not stable release the support for Windows

**Comment by timneutkens** [comment]
We've been very successful with Azure Pipelines on Next.js:
https://azure.microsoft.com/en-us/services/devops/pipelines/

I think now-desktop also uses it.

**Comment by amio** [comment]
Azure Pipelines looks promising, let's give it a try 👍

Two PRs for this:

1. Get build script compatible with windows (#12)
2. Update azure pipelines config for testing with windows image (#13)

**Comment by MAPESO** [comment]
@amio If we are going to use **Azure pipeline** I recommend deleting the `add/appveyor-configuration` branch. I see no case that continues to exist 👀

<img width="688" alt="Screen Shot 2019-04-06 at 00 31 49" src="https://user-images.githubusercontent.com/16585386/55665949-c7666200-5804-11e9-8924-c29076891282.png">

**Comment by amio** [comment]
@MAPESO Yep, branch deleted. I also disabled appveyor webhook, no more alert from appveyor now 😎

**Comment by MAPESO** [comment]
@amio Great !! 😺
